source/plugins/go/src/oms.go

```diff
@@ -1870,6 +1968,17 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 	ContainerLogSchemaV2 = false //default is v1 schema
 	ContainerLogV2ConfigMap = (strings.Compare(ContainerLogSchemaVersion, ContainerLogV2SchemaVersion) == 0)
 
+	KubernetesMetadataConfigMap = false
+	KubernetesMetadataConfigMap = (strings.Compare(strings.ToLower(os.Getenv("AZMON_KUBERNETES_METADATA_ENABLED")), "true") == 0)
+	metadataIncludeList := os.Getenv("AZMON_KUBERNETES_METADATA_INCLUDES_FIELDS")
+	Log(fmt.Sprintf("KubernetesMetadataIncludeList from configmap: %+v\n", metadataIncludeList))
+	KubernetesMetadataIncludeList = []string{"podLabels", "podAnnotations", "podUid", "image"}
+	if KubernetesMetadataConfigMap && len(metadataIncludeList) > 0 {
+		KubernetesMetadataIncludeList = strings.Split(metadataIncludeList, ",")
+	} else if KubernetesMetadataConfigMap {
```

Comment:
if a customer does not provide include_list that is let's say they enabled it via
```

       [log_collection_settings.metadata_collection]
          # if enabled will collect kubernetes metadata for ContainerLogv2 schema. Default false
          enabled = true
          # include_fields = ["podLabels","podAnnotations","podUid","image"]
```

what will the value of metadataIncludeList be in this case?

Reply:
the default value will be "podLabels","podAnnotations","podUid","image"